src/wallet/test/wallet_tests.cpp

```diff
@@ -73,6 +73,27 @@ static void AddKey(CWallet& wallet, const CKey& key)
     assert(spk_manager);
 }
 
+BOOST_FIXTURE_TEST_CASE(update_non_range_descriptor, TestingSetup)
+{
+    CWallet wallet(m_node.chain.get(), "", CreateMockableWalletDatabase());
+    {
+        LOCK(wallet.cs_wallet);
+        wallet.SetWalletFlag(WALLET_FLAG_DESCRIPTORS);
+        auto key{GenerateRandomKey()};
+        auto desc_str{"combo(" + EncodeSecret(key) + ")"};
+        for (size_t i = 0; i < 2; i++)
```

Comment:
Nit: Although the test name is present but it might be helpful to mention why the loop is there, i.e., to update the descriptor by calling `AddWalletDescriptor` multiple times with the same descriptor.

Reply:
Decided to simplify the test. See https://github.com/bitcoin/bitcoin/pull/32344/commits/97d383af6d54b463da64f45680a146d7e93eb146